apps/cli/config/config.exs

```diff
@@ -2,29 +2,18 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
-# This configuration is loaded before any dependency and is restricted
-# to this project. If another project depends on this project, this
-# file won't be loaded nor affect the parent project. For this reason,
-# if you want to provide default values for your application for
-# 3rd-party users, it should be done in your "mix.exs" file.
+config :cli,
+  cookie:
+    :erlang.apply(
```

Comment:
Just curious, is there a reason why we can't use elixir's [`apply/2`](https://hexdocs.pm/elixir/Kernel.html#apply/2) function here instead of `:erlang.apply/2`?

Reply:
technically it's erlang's apply either way, but I get your point